go.mod

```diff
@@ -1,14 +1,14 @@
 module github.com/celestiaorg/celestia-node
 
-go 1.22.4
+go 1.22.6
```

Comment:
Does go version need to be bumped?

Reply:
yes

Reply:
app is on 1.22.6 

`
packages.Load error: err: exit status 1: stderr: go: [github.com/celestiaorg/celestia-app/v2@v2.1.0](http://github.com/celestiaorg/celestia-app/v2@v2.1.0) requires go >= 1.22.5 (running go 1.22.4)go list`

Reply:
Go 1.23 is out. Lets bump to latest version in different PR

Reply:
Will bump with this PR:
https://github.com/celestiaorg/celestia-node/pull/3676

Reply:
sg! we need to release a patch anyway so I want to hold off on merging this till then.

Reply:
resolving this